### PR TITLE
Community council update

### PR DESCRIPTION
**Pull request by Sander3003**
Removed me from the community council / TAC

### CODE REVIEW

**Comment by Sander3003** [comment]
"Security issues can be reported on the one of the maintainers personally." is this correct?

File: Opensourcecommunity/Code-of-conduct.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ Diversity is one of our huge strengths, but it can also lead to communication is
...
 
-This code of conduct applies to all spaces managed by the open smart grid platform project (or Open Smart Grid Platform Software Foundation once established). This includes IRC, the mailing lists, the issue tracker, DSF events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+This code of conduct applies to all spaces managed by the GXF project (a LFenergy project). This includes IRC, the mailing lists, the issue tracker, DSF events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
 
-If you believe someone is violating the code of conduct, we ask that you report it by emailing info@opensmartgridplatform.org. 
+If you believe someone is violating the code of conduct, we ask that you report it by emailing the maillinlist gxf@lists.lfenergy.org.
```
Comment (smvdheijden) [inline Opensourcecommunity/Code-of-conduct.md:11]: maillinlist -> mailing list

File: Opensourcecommunity/Communication-and-contact.md [hunk 2]
```diff
@@ -3,15 +3,15 @@
 We choose [Jira](https://smartsocietyservices.atlassian.net/projects/OC/issues/) for most of our development communication. This keeps the communication central and topic central and connect to development effort. 
 Please use [Jira](https://smartsocietyservices.atlassian.net/projects/OC/issues/) for issues and bugs. 
 
-If you want to get in touch to discuss non-technical subjects, send us an email info@opensmartgridplatform.org or open an [issue on Github](https://github.com/OSGP/documentation/issues/new?title=Question%20:My%20Title&body=**Question:**%0A%0A**background:**%0A%0A**).
+If you want to get in touch to discuss non-technical subjects, send us an email to the LF energy GXF maillinglist gxf@lists.lfenergy.org or open an [issue on Github](https://github.com/OSGP/documentation/issues/new?title=Question%20:My%20Title&body=**Question:**%0A%0A**background:**%0A%0A**).
```
Comment (smvdheijden) [inline Opensourcecommunity/Communication-and-contact.md:6]: maillinglist -> mailing list

File: Opensourcecommunity/Code-of-conduct.md [hunk 3]
```diff
@@ -6,9 +6,9 @@ Diversity is one of our huge strengths, but it can also lead to communication is
 
 This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
 
-This code of conduct applies to all spaces managed by the open smart grid platform project (or Open Smart Grid Platform Software Foundation once established). This includes IRC, the mailing lists, the issue tracker, DSF events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+This code of conduct applies to all spaces managed by the GXF project (a LFenergy project). This includes IRC, the mailing lists, the issue tracker, DSF events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
```
Comment (smvdheijden) [inline Opensourcecommunity/Code-of-conduct.md:9]: LFenergy -> LF Energy

File: Opensourcecommunity/Communication-and-contact.md [hunk 4]
```diff
@@ -37,4 +37,4 @@ If you have a question, please create an Github issue in the [documentation repo
 
  
 ### Report security issues
-Due to the sensible nature of security issues e.g. zero days, we prefer a responsible disclosure. Security issues can be reported on the [opensmartgridplatform.org website](https://opensmartgridplatform.org/report-security-issue/).
+Due to the sensible nature of security issues e.g. zero days, we prefer a responsible disclosure. Security issues can be reported on the one of the maintainers personally.
```
Comment (smvdheijden) [inline Opensourcecommunity/Communication-and-contact.md:40]: can be reported on the one of the maintainers personally -> can be reported at one of the maintainers personally

File: Opensourcecommunity/Governance.md [hunk 5]
```diff
@@ -19,7 +19,7 @@ The community council is responsible for:
 When the community grows, members of the community council can be elected. If the situation demands or requires it, the community council has the ability to establish sub councils for a specific subject, area of domain.
 
 The community council consist of the following members:
-* [Sander Jansen](https://github.com/Sander3003) - Open source consultant - Chairman 
+* [Robert Tusveld](https://github.com/Sander3003) - Architect - Chairman
```
Comment (smvdheijden) [inline Opensourcecommunity/Governance.md:22]: Replace the github account (Sander3003) as well.

**Review by smvdheijden — CHANGES_REQUESTED** [review]

File: Opensourcecommunity/Code-of-conduct.md (hunk 1 above)
Comment (Sander3003) [reply Opensourcecommunity/Code-of-conduct.md]: Thanks, fixed.

File: Opensourcecommunity/Code-of-conduct.md [hunk 6]
```diff
@@ -6,9 +6,9 @@ Diversity is one of our huge strengths, but it can also lead to communication is
...
 
-This code of conduct applies to all spaces managed by the GXF project (a LFenergy project). This includes IRC, the mailing lists, the issue tracker, DSF events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+This code of conduct applies to all spaces managed by the GXF project (a LF energy project). This includes IRC, the mailing lists, the issue tracker, DSF events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
 
-If you believe someone is violating the code of conduct, we ask that you report it by emailing the maillinlist gxf@lists.lfenergy.org. 
+If you believe someone is violating the code of conduct, we ask that you report it by emailing the mailling list gxf@lists.lfenergy.org.
```
Comment (smvdheijden) [inline Opensourcecommunity/Code-of-conduct.md:11]: mailing is written with a single "l"

File: Opensourcecommunity/Communication-and-contact.md [hunk 7]
```diff
@@ -3,7 +3,7 @@
 We choose [Jira](https://smartsocietyservices.atlassian.net/projects/OC/issues/) for most of our development communication. This keeps the communication central and topic central and connect to development effort. 
 Please use [Jira](https://smartsocietyservices.atlassian.net/projects/OC/issues/) for issues and bugs. 
 
-If you want to get in touch to discuss non-technical subjects, send us an email to the LF energy GXF maillinglist gxf@lists.lfenergy.org or open an [issue on Github](https://github.com/OSGP/documentation/issues/new?title=Question%20:My%20Title&body=**Question:**%0A%0A**background:**%0A%0A**).
+If you want to get in touch to discuss non-technical subjects, send us an email to the LF energy GXF mailling list gxf@lists.lfenergy.org or open an [issue on Github](https://github.com/OSGP/documentation/issues/new?title=Question%20:My%20Title&body=**Question:**%0A%0A**background:**%0A%0A**).
```
Comment (smvdheijden) [inline Opensourcecommunity/Communication-and-contact.md:6]: mailing

**Review by smvdheijden — CHANGES_REQUESTED** [review]

**Comment by Sander3003** [comment]
Bugs are fixed

**Comment by Jonasvdbo** [comment]
@Sander3003 In regards to your question: "Security issues can be reported on the one of the maintainers personally. Is this correct? And is this the way to do it in the LF energy community?" There is no best practices in LF energy community for this yet. I would say security issues can be reported to gxf+owner@lists.lfenergy.org

**Review by smvdheijden — APPROVED** [review]

**Pull request merged**